edx_repo_tools/audit_gh_users/audit_users.py

```diff
@@ -64,8 +65,33 @@ def main(org, _github_token, csv_repo, csv_path):
     # Find all the people that are in the org but not in sales force.
     extra_org_users = set(current_org_users) - set(csv_github_users)
 
+    # Find users who are in multiple teams or a single non-triage team
```

Comment:
```suggestion
    # Find users who are in multiple teams or a single non-triage team
    # Using the GraphQL API because there is no good GitHub rest API for this.
```